admin/data/sql/migrations/20200127112400.do.alter-view-vewpupilswithactivelivepins.sql

```diff
@@ -0,0 +1,24 @@
+ALTER VIEW [mtc_admin].[vewPupilsWithActiveLivePins] AS
```

Comment:
Just to follow the strategy that we have for other migrations shouldn't that be `CREATE OR ALTER` ?

Reply:
Good point.  Amended.  Thanks.